src/data/3.x/guides.js

```diff
@@ -1,94 +1,94 @@
-// guides are the pages under the "Documentation" tab
+// guides are the pages under the Documentation" tab
 
 const guides = {
     pages: {
         "install-dist": {
-            path: "install-dist",
-            title: "1. Installation",
+            path: `install-dist`,
+            title: `1. Installation`,
         },
         "include-scripts": {
-            path: "include-scripts",
-            title: "2. Including Scripts",
+            path: `include-scripts`,
+            title: `2. Including Scripts`,
         },
         "configuration": {
-            path: "configuration",
-            title: "3. Configuration",
+            path: `configuration`,
+            title: `3. Configuration`,
         },
         "your-first-search": {
-            path: "your-first-search",
-            title: "4. Your First Search",
+            path: `your-first-search`,
+            title: `4. Your First Search`,
         },
         "upgrade-guide": {
             path: "upgrade-guide",
             title: "5. Upgrade Guide 1.2.x to 3.0.0",
```

Comment:
Not changing " -> ` here?

Reply:
Ah! That one didn't exist yet in the original PR 😄